workflow/controller/controller.go

```diff
@@ -417,6 +421,17 @@ func (wfc *WorkflowController) processNextItem() bool {
 	return true
 }
 
+func (wfc *WorkflowController) addingWorkflowDefaultValueIfValueNotExist(wf *wfv1.Workflow) {
+	//var workflowSpec *wfv1.WorkflowSpec = &wf.Spec
+	if wfc.Config.WorkflowDefaults != nil {
+		if err := mergo.Merge(&wf.Spec, *wfc.Config.WorkflowDefaults); err != nil {
+			log.Errorf("Failed to merge defaults and workflow : %+v", err)
+			panic(err)
```

Comment:
never panic!

Reply:
What is the best practice, just log it and return the error?

Reply:
Return error